documentation/3_Interoperability.md

```diff
@@ -0,0 +1,375 @@
+fragmentation and duplication of efforts, participants in these
+Data Spaces need to communicate in an interoperable way with each other
+and across multiple Data Spaces, following common standards and
+principles.
+
+Interoperability can be achieved at different levels, depending on the
+degree of integration and alignment of the data and systems involved.
+Two well-known frameworks that define interoperability levels are the
+[ISO/IEC
+19941](https://standards.iso.org/ittf/PubliclyAvailableStandards/c079573_ISO_IEC_19944-1_2020(E).zip)
+standard for cloud computing interoperability and portability, and the
+European Interoperability Framework for public services. Both frameworks
+identify four main levels of interoperability: technical (transport &
+syntactic), semantic, organizational, and legal:
+
+- Technical interoperability refers to the physical and logical
+    connections between systems and data sources, such as protocols,
+    interfaces, and formats. This includes syntactic interoperability
+    which refers to the structure and syntax of the data exchanged, such
+    as schemas, models, and vocabularies.
+
+- Semantic interoperability refers to the meaning and interpretation
+    of the data, such as concepts, relationships, and ontologies.
+
+- Organizational interoperability refers to the processes, policies,
+    and governance of data sharing, such as roles, responsibilities, and
+    agreements.
+
+- Legal interoperability refers to the acceptance of legal equivalence
+    of contracts and contractual clauses between different data
+    ecosystems. These ecosystems can have differences on multiple
+    dimensions, based for example on industry regulations, or national
+    laws but also contractual statements with identical wordings might
+    have diverging interpretations in different data ecosystems.
+
+In this chapter, we discuss the challenges and opportunities of
+achieving inter- and intra- Data Space interoperability at these levels,
+and propose a roadmap for developing a common framework and best
+practices for Data Spaces.
+
+## Guiding principles for Data Spaces
+
+As described in the previous sections, there are guiding principles in
+Data Spaces that are not only shaping the functional requirements, but
+also take effect in reasoning over interoperability. Those guiding
+principles are the foundation of any interoperability framework for
+Data Spaces. Let's have a closer look at those fundamental principles
+again:
+
+1. Self-determined control of data use (Data Sovereignty as also part of [ISO/IEC CD TS 10866, Framework and concepts for organizational autonomy and digital sovereignty](https://www.iso.org/standard/83757.html) ) is of
+    utmost importance and should be the ideal vision that each Data Space
+    thrives to enable.
+
+    1. Participants have autonomy and are able to act with choice
+
+    2. Participants have agency over their data assets
+
+2. A Data Space creates a context of trust
```

Comment:
Please add: "trust does tackle technical as well as semantical layers".

Reply:
not quite sure what you mean and how you want to express this. Can you provide the entire sentence or elaborate more?

Reply:
btw, trust is the purpose of the dataspace and thus covers all 4 layers: Technical, Semantic, Process and Legal. 
It would be redundant to explain the the trust context created by a dataspace needs to include all 4 layers.